cameraViewEx/src/main/api21/com/priyankvasa/android/cameraviewex/VideoConfiguration.kt

```diff
@@ -65,6 +68,9 @@ class VideoConfiguration {
         const val BIT_RATE_1080P = 16000000
         const val BIT_RATE_MIN = 64000
         const val BIT_RATE_MAX = 40000000
+
+        const val DEFAULT_MAX_DURATION = 0
```

Comment:
I think this value should be a sensible greater value say 60 mins (3600000) so that while checking
```
config.maxDuration >= VideoConfiguration.DEFAULT_MIN_DURATION
```
A warning can be thrown to user if the set `maxDuration` is not valid.

Reply:
Updated to 60 minutes